global.json

```diff
@@ -0,0 +1,10 @@
+{
+  "sdk": {
```

Comment:
Do we really need to pin the SDK version?

Reply:
It's a good practice for more build determinism.

Reply:
There's no need to pin the SDK version as it's already pinned by the explicit SDKs installed by uno-check. More that anything, it's generating friction for local developers that may not have the exact version installed.

Reply:
Looking further, it will create discrepancies because the version specified in uno-check are not aligned with the .NET version installed by uno-check (it's still 6.0.40x).

Note also that we still need to wait for .NET 7 SR1 to be released.

Reply:
> it's generating friction for local developers that may not have the exact version installed.

Updating the SDK version doesn't happen very often, and it's too easy to update when it happens.